package-res/resources/web/pentaho/data/filter/And.js

```diff
@@ -24,13 +24,13 @@ define([
    * @name And
    * @memberOf pentaho.data.filter
    * @class
-   * @abstract
+   * @extends pentaho.data.filter.AbstractTreeFilter
    * @amd pentaho/data/filter/And
    *
    * @classdesc The `And` class implements a type of AbstractTreeFilter {@link pentaho.data.filter.AbstractTreeFilter}.
```

Comment:
Not like this. This description also applies to Or. What is the difference between an And and an Or? One performs intersection, the other the union of two sets.